### PR TITLE
Release version 1.1.5

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
**Summary:** Release 1.1.5

* Update packages to 1.1.5
* Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +44,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - `@cumulus/deployment` deploys DynamoDB streams for the Collections, Providers and Rules tables as well as a new lambda function called `dbIndexer`. The `dbIndexer` lambda has an event source mapping which listens to each of the DynamoDB streams. The dbIndexer lambda receives events referencing operations on the DynamoDB table and updates the elasticsearch cluster accordingly.
   - The `@cumulus/api` endpoints for collections, providers and rules _only_ query DynamoDB, with the exception of LIST endpoints and the collections' GET endpoint.
 
+- **CUMULUS-260: "PDR page on dashboard only shows zeros."** The PDR stats in LPDAAC are all 0s, even if the dashboard has been fixed to retrieve the correct fields.  The current version of pdr-status-check has a few issues.
```
Comment (abarciauskas-bgse) [inline CHANGELOG.md]: Hmm @laurenfrederick should this be in unreleased?

**Comment by sethvincent** [comment]
I think this looks good. I'm not sure why the integration tests are failing after updating the package version.

If 585920b fixes the cloudwatchevents issue we should just close #258.

**Comment by abarciauskas-bgse** [comment]
@sethvincent thanks for taking a look - integration tests are failing and it has to do with the KMS error that Menno was seeing. I have verified that the HelloWorldWorkflow succeeds if we update all the lambda encryptions to to use `aws/lambda` instead of `(default) aws/lambda` and I'm just trying to get the stack deployed so all functions are updated. Seems like something with the KMS keys in AWS must have changed.

**Comment by abarciauskas-bgse** [comment]
Updated circleci cumulus project to use `abarciauskas-bgse_testing-359` branc of `cumulus-integration-tests`, see: https://github.com/cumulus-nasa/cumulus-integration-tests/pull/8

**Comment by abarciauskas-bgse** [comment]
Successfully terminated the `test-cumulus` stack, so re-running circleci using the master branch of `cumulus-integration-tests`. This should 🤞 re-create the `test-cumulus` stack and I will close https://github.com/cumulus-nasa/cumulus-integration-tests/pull/8.

File: CHANGELOG.md (hunk 1 above)
Comment (laurenfrederick) [reply CHANGELOG.md]: Yes

**Comment by abarciauskas-bgse** [comment]
Closing in need of minor release https://github.com/cumulus-nasa/cumulus/pull/265

**Pull request closed without merging**